packages/analytics-js-integrations/package.json

```diff
@@ -105,6 +105,7 @@
     "@rudderstack/analytics-js-common": "*",
     "component-each": "0.2.6",
     "crypto-js": "4.2.0",
+    "each": "^2.6.0",
```

Comment:
what is this new library? Do we need it? 
does https://ramdajs.com/docs/#forEach cover our needs?

Reply:
if we do we should not allow ^ range and it should also be in root level package json